salt/config/__init__.py

```diff
@@ -351,9 +357,9 @@ def _gather_buffer_space():
         "test": bool,
         # Tell the loader to attempt to import *.pyx cython files if cython is available
         "cython_enable": bool,
-        # Whether or not to load grains for FQDNs
+        # Whether or not to load grains for FQDNs (deprecated)
```

Comment:
Given that the schema can have extra options, is it ok to remove the settings from it?

Reply:
I think its okay to put them on a deprecation path in favor of the new disabled_grains.

Can you make sure you add a comment:
`# salt.utils.versions.warn_until(Potassium, "enabled_fqdns_grains is deprecated in favor of.......)`
The only reason being is we grep the cod for warn_until and version name each time we work on removing deprecations.

Also add changelog files for each setting you are removing with the filename deprecated stating the details on what users can now use instead.